tests/rbd_mirror/test-image-delete-from-secondary.py

```diff
@@ -1,58 +1,20 @@
 import datetime
 import time
```

Comment:
please removed unused imports? `time` and `datetime` no where used in this module.

Reply:
time and datetime is used in line no 22 and 54